### PR TITLE
#477 Migrate React Components to ES6 syntax

### PR DESCRIPTION
**Pull request by Sudip7**

### CODE REVIEW

**Comment by Sudip7** [comment]
updated some file for : [#477](https://github.com/mozilla/network-pulse/issues/477)

**Comment by alanmoo** [comment]
Thanks for the contribution! @mmmavis is out this week, so it might be a little bit before she gets to review this.

**Comment by Sudip7** [comment]
@alanmoo, thanks for the update. Do u guys have any new communication channel i.e. slack or anything?? I have not heard from u all for quite some time in the Mattermost regular network pulse channel. Please update. @mmmavis, @pomax

File: components/issue-selector/issue-selector.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import NavLink from '../nav-link/nav-link.jsx';
 
-export default React.createClass({
+class IssueSelector extends React.Component{
```
Comment (Pomax) [inline components/issue-selector/issue-selector.jsx]: please put a space between `Component` and `{` for all three classes

File: components/issue-selector/issue-selector.jsx [hunk 2]
```diff
@@ -13,4 +13,5 @@ export default React.createClass({
       </div>
     );
   }
-});
+}
+export default IssueSelector
```
Comment (Pomax) [inline components/issue-selector/issue-selector.jsx]: missing semicolon

**Comment by Sudip7** [comment]
@Pomax @mmmavis please review

**Comment by mmmavis** [comment]
I'm back from PTO. Will review this today.

**Pull request merged**